src/components/Footer/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Link from '@material-ui/core/Link';
```

Comment:
combine imports, try 
```suggestion
import { Typography, Link } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';
```